native/cocos/audio/android/AudioEngine-inl.h

```diff
@@ -25,7 +25,11 @@ THE SOFTWARE.
 #pragma once
 
 #include <SLES/OpenSLES.h>
+#if CC_PLATFORM == CC_PLATFORM_ANDROID
 #include <SLES/OpenSLES_Android.h>
+#elif CC_PLATFORM == CC_PLATFORM_OPENHARMONY
+#include <SLES/OpenSLES_Platform.h>
```

Comment:
If we would like to distinguish the header file included, why don't we create a new header for all headers?

```cpp
// In a new file CommonOpenSLES.h or some other name
#if CC_PLATFORM == CC_PLATFORM_ANDROID
#include <SLES/OpenSLES_Android.h>
#elif CC_PLATFORM == CC_PLATFORM_OPENHARMONY
#include <SLES/OpenSLES_Platform.h>
#endif
```

Reply:
It's not that most places are different, but a few files are different, and there are not many places where we are used. There is no need to create such a header file, which is somewhat redundant.